src/js/edu-benefits/components/education-history/EducationHistoryFields.jsx

```diff
@@ -35,29 +35,29 @@ export default class EducationHistoryFields extends React.Component {
             year={year}
             onValueChange={(update) => {this.props.onStateChange('highSchoolOrGedCompletionDate', update);}}/>
       </div>
-      {isValidDate(day.value, month.value, year.value)
```

Comment:
When I looked at the wireframes, the table wasn't conditional. I can remove this change if it is actually supposed to be conditional.